certora/specs/SocialRecoveryModule.spec

```diff
@@ -19,24 +19,46 @@ methods {
     // Safe Functions
     function safeContract.isModuleEnabled(address module) external returns (bool) envfree;
     function safeContract.isOwner(address owner) external returns (bool) envfree;
+    function safeContract.getOwners() external returns (address[] memory) envfree;
+    function safeContract.getThreshold() external returns (uint256) envfree;
 
     // Wildcard Functions (Because of use of ISafe interface in Social Recovery Module)
     function _.isModuleEnabled(address module) external => summarizeSafeIsModuleEnabled(calledContract, module) expect bool ALL; // `calledContract` is a special variable.
     function _.isOwner(address owner) external => summarizeSafeIsOwner(calledContract, owner) expect bool ALL;
+    function _.getOwners() external => summarizeSafeGetOwners(calledContract) expect address[] ALL;
+    function _.execTransactionFromModule(address to, uint256 value, bytes data, Enum.Operation operation) external with (env e) => summarizeSafeExecTransactionFromModule(calledContract, e, to, value, data, operation) expect bool ALL;
```

Comment:
There is a more succinct way of expressing this instead of manually writing out summaries so that the prover can resolve the calls:

https://github.com/5afe/CandideWalletContracts/blob/16a9978da3634992d8f6b691d7a01ccd339f8d8e/certora/specs/SocialRecoveryModule.spec#L28-L30

Reply:
Ah, this looks like some rebasing/merge issue and not a change by you (judging by the rule introduced in this file).